lib/tasks/geoblacklight.rake

```diff
@@ -0,0 +1,12 @@
+require 'geoblacklight/dataingest'
+
+namespace :geoblacklight do
```

Comment:
The geoblacklight gem has a `geoblacklight` name space, so maybe we should prefix ours to ensure we don't get namespace collisions in the future?  Perhaps we could have `vtul:geoblacklight` or `dld:geoblacklight` or even `vtul:dld:geoblacklight`?  Anyway, I think we should avoid using `geoblacklight` as a top-level Rake name space.